include/fcl/octree.h

```diff
@@ -97,6 +97,12 @@ class OcTree : public CollisionGeometry
     aabb_radius = (aabb_local.min_ - aabb_center).length();
   }
 
+  /// @brief get the underlying octree structure
+  boost::shared_ptr<octomap::OcTree> getTree()
+  {
+    return tree;
+  }
```

Comment:
It seems this accessor is added to address #73, which looks good to me. It would also good to provide const version of `getTree()` for const correctness as:
```cpp
boost::shared_ptr<const octomap::OcTree> getTree() const
{
  return tree;
}
```